tests/test_utils.py

```diff
@@ -1597,3 +1597,61 @@ def test_union(objs, expected):
         audformat.utils.union(objs),
         expected,
     )
+
+
+@pytest.mark.parametrize(
+    'obj, expected',
+    [
+        (
+            pd.Series(),
+            []
```

Comment:
```suggestion
            [],
```

We always end multi-line arguments with a comma after the last entry. Please also add a comma in the other cases.

Reply:
Actually, to match how we usually format our code, please change to:

```python
@pytest.mark.parametrize(
    'obj, expected',
    [
        (
            pd.Series(),
            [],
        ),
        (
            audformat.filewise_index(),
            [],
        ),
        (
            audformat.segmented_index(
                ['f1', 'f1', 'f2'],
                [0, 1, 0],
                [2, 3, 1],
            ),
            [
                (
                    'f1', 
                    audformat.segmented_index(
                        ['f1', 'f1'], 
                        [0, 1],
                        [2, 3],
                    ),
                ),
                (
                    'f2', 
                    audformat.segmented_index(['f2'], [0], [1]),
                ),
            ],
        ),
        (
            pd.Series(
                index=audformat.segmented_index(
                    ['f1', 'f1', 'f2'],
                    [0, 1, 0], 
                    [2, 3, 1],
                ),
            ),
            [
                (
                    'f1', 
                    pd.Series(
                        index=audformat.segmented_index(
                            ['f1', 'f1'],
                            [0, 1], 
                            [2, 3],
                        ),
                    ),
                ),
                (
                    'f2', 
                    pd.Series(
                        index=audformat.segmented_index(
                            ['f2'],
                            [0], 
                            [1],
                        ),
                    ),
                ),
            ],
        ),
        (
            pd.DataFrame(
                index=audformat.filewise_index(['f1', 'f1', 'f2']),
            ),
            [
                (
                    'f1', 
                    pd.DataFrame(
                        index=audformat.filewise_index(['f1', 'f1']),
                    ),
                ),
                (
                    'f2', 
                    pd.DataFrame(
                        index=audformat.filewise_index(['f2']),
                    ),
                ),
            ],
        ),
    ]
)
```